fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -3291,6 +3317,36 @@ public void testResponseContentTypes() throws Exception {
         }
     }
 
+    /**
+     * Without the asterisks on the GET/HEAD methods Jersey will fail to serve all other content types for binaries.
+     */
+    @Test
+    public void testBinaryAcceptHeaders() throws IOException {
+        final HttpPost post = postObjMethod();
+        post.setHeader(CONTENT_TYPE, "application/pdf");
+        post.setEntity(new StringEntity("Some fake content", UTF_8));
+        final String objectUrl;
+        try (final CloseableHttpResponse response = execute(post)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+            objectUrl = getLocation(response);
+        }
+        final HttpGet httpGet = new HttpGet(objectUrl);
+        httpGet.setHeader(ACCEPT, "image/tiff");
+        assertEquals(NOT_ACCEPTABLE.getStatusCode(), getStatus(httpGet));
+
+        final HttpGet httpGet2 = new HttpGet(objectUrl);
+        httpGet2.setHeader(ACCEPT, "application/pdf");
+        assertEquals(OK.getStatusCode(), getStatus(httpGet2));
+
+        final HttpHead httpHead = new HttpHead(objectUrl);
+        httpHead.setHeader(ACCEPT, "image/tiff");
+        assertEquals(NOT_ACCEPTABLE.getStatusCode(), getStatus(httpHead));
+
+        final HttpHead httpHead2 = new HttpHead(objectUrl);
+        httpHead2.setHeader(ACCEPT, "application/pdf");
+        assertEquals(OK.getStatusCode(), getStatus(httpHead2));
+    }
```

Comment:
Might be good to test in here what happens when `Accept: */*` is supplied, and no Accept header is provided.